python/gtsam/utils/plot.py

```diff
@@ -108,6 +108,69 @@ def plot_covariance_ellipse_3d(axes,
     axes.plot_surface(x, y, z, alpha=alpha, cmap='hot')
 
 
+def plot_point2_on_axes(axes,
+                        point: Point2,
+                        linespec: str,
+                        P: Optional[np.ndarray] = None) -> None:
+    """
+    Plot a 2D point on given axis `axes` with given `linespec`.
+
+    Args:
+        axes (matplotlib.axes.Axes): Matplotlib axes.
+        point: The point to be plotted.
+        linespec: String representing formatting options for Matplotlib.
+        P: Marginal covariance matrix to plot the uncertainty of the estimation.
+    """
+    axes.plot([point[0]], [point[1]], linespec, marker='.', markersize=10)
+    if P is not None:
+        w, v = np.linalg.eig(P)
+
+        # "Sigma" value for drawing the uncertainty ellipse. 5 sigma corresponds
```

Comment:
@magicbycalvin This is not correct in 2D, your values seem to be for the 1D case. See `Geometric interpretation` [here](https://en.wikipedia.org/wiki/Multivariate_normal_distribution#Properties) and read [this](https://stackoverflow.com/a/39749274/1959116). Can you fix that in another PR?

Reply:
Actually, I added a note in #1063, and I think k=5 indeed corresponds to 99.9996% in 2D case. Check the note and let me know... That PR could be amended to accommodate a change if needed, if @gchenfc does not merge it first.

Reply:
It's in plot.py, in that PR

Reply:
Just now seeing this discussion. I have since added an additional PR addressing the comments above.

Reply:
> Actually, I added a note in #1063, and I think k=5 indeed corresponds to 99.9996% in 2D case. Check the note and let me know... That PR could be amended to accommodate a change if needed, if @gchenfc does not merge it first.

While I agree with 99.9996% (see #1067), that is not what was written in the comment:

> to a 99.9999% confidence, i.e. assuming the estimation has been computed properly, there is a 99.999% chance that the true position